dicom/config_service_import_service.go

```diff
@@ -0,0 +1,53 @@
+package dicom
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type ImportService struct {
+	ID      string `json:"id,omitempty"`
```

Comment:
There is also a description field

Reply:
Ah, this one has bitten me before. the HSDP documentation sometimes add a `description:` text that looks very much like an actual Model field. This is also the case for the `configData` description. https://www.hsdp.io/documentation/dicom-store/api-documents/configuration-api#/ImportService/CreateOrUpdateImportService

Reply:
I know